dsa-23au/java-dsa/NMac-app/src/Driving_Analysis.java

```diff
@@ -0,0 +1,83 @@
+// The purpose of this program:
+// Using the CSV file
+// Print out the table
+// Calculate the total elapsed time driving divided by the total distance
+
+// So far the program does not read the csv file correctly.
+// The program is reading driver although set to read the distance
+// Not sure if the elapsed time being double minutes will conflict with it assumes to be in hours
+
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Driving_Analysis {
+
+    public static class DrivingRecord {
+        String time;
+        String date;
+        double elapsed; // Assumes this is in hours // the csv data is in minutes... that could be a problem
```

Comment:
https://stackoverflow.com/questions/40285248/java-time-difference-in-minutes
https://stackoverflow.com/questions/1770010/how-do-i-measure-time-elapsed-in-java
use System.nanoTime() for precision
https://mkyong.com/java/java-time-elapsed-in-days-hours-minutes-seconds/
Everything looks great so far